public/static/js/projects.js

```diff
@@ -278,10 +280,14 @@ mciModule.controller('ProjectCtrl', function($scope, $window, $http, $location,
           return v.alias + v.variant + v.task;
         });
 
-        // Divide aliases into two categories (gh/patch aliases)
-        [$scope.github_aliases, $scope.patch_aliases] = _.partition(
-          $scope.aliases, function(d) { return d.alias == '__github' }
+        // Divide aliases into three categories (patch, github, and commit queue aliases)
+        $scope.github_aliases = $scope.aliases.filter(
+          function(d) { return d.alias == '__github' }
         )
+        $scope.commit_queue_aliases = $scope.aliases.filter(
+          function(d) { return d.alias == '__commit_queue' }
+        )
+        $scope.patch_aliases = _.difference($scope.aliases, $scope.github_aliases.concat($scope.commit_queue_aliases))
```

Comment:
Could we do _.filter with alias !== either of the ones above? The current method seems confusing to me

Reply:
done